.travis.yml

```diff
@@ -7,7 +7,7 @@ env:
     - CODENAME: reblochon
   matrix:
     - DOCKER_VERSION=1.9.1
-    - DOCKER_VERSION=1.10.1
+    - DOCKER_VERSION=1.12
```

Comment:
```https://get.docker.com/builds/Linux/x86_64/docker-${DOCKER_VERSION}``` replies 404 when applied with 1.12 and circle-ci fail on that error :-(

Could you:
- add a ```.tgz``` after ```docker-${DOCKER_VERSION}````
- set ```DOCKER_VERSION``` to ```1.12.0```